android/assets/jsons/Civ V - Gods & Kings/TileImprovements.json

```diff
@@ -176,8 +176,17 @@
 		"uniques": ["Can be built outside your borders"],
 		"shortcutKey": "."
 	},
+    {
+        "name": "Repair",
+        "terrainsCanBeBuiltOn": ["Land"],
+        "turnsToBuild": 2,
+        "uniques": ["Can be built outside your borders"],
```

Comment:
Hmmm I'm not 100% sure about this since this could lead to pillaging and repairing enemy improvements, I'm okay with it for now but we need to keep this in mind

Reply:
Hm, I didn't check, but can you pillage neutral improvements?

I can add a check so you can't repair Enemy Improvements (not sure why you'd want to except prepping for a city takeover)

Reply:
There's a "Pillaging this improvement yields [stats]" unique, and if you can repair enemy improvements that can lead to you squatting on an enemy tile with a military unit and a worker and pillage/repairing it every 2 turns

Reply:
OK, I'll drop it then